src/client/views/images/index.js

```diff
@@ -9,25 +9,44 @@ import { DropDown } from './DropDown'
 
 import './index.scss'
 
-function Images ({ images }) {
-  return (
-    <div className="images">
-    <DropDown title='Image Layout'/>
-      <SEO title="Images" />
-      <FileLoader
-        className="image-files"
-        files={images}
-        loader={fetchImages}
-        display={(image, key) => (
-          <div className="image" key={key}>
-            <a href={image.name.split('.')[0]} target="_blank" >
-              <img src={image.name} />
-            </a>
-          </div>
-        )}
-      />
-    </div>
-  )
+class Images extends React.Component {
```

Comment:
this should probably still be a functional component for consistency
use hooks for state `useState`